README.md

```diff
@@ -75,7 +75,7 @@ command to find them out:
 $ crio config |& grep -B1 -A5 cdi_spec_dirs
 ```
 
-### Containerd configuration
+### containerd configuration
 
 To enable and configure CDI support in the [containerd
```

Comment:
how about adding? i ended up looking for this information to try out, which would be useful for other uses?

```suggestion
[cotnainerd v1.7.0](https://github.com/containerd/containerd/releases/tag/v1.7.0) or later required.
To enable and configure CDI support in the [containerd
```